src/App.vue

```diff
@@ -1,85 +1,136 @@
-<script setup>
-import { RouterLink, RouterView } from 'vue-router'
-import HelloWorld from './components/HelloWorld.vue'
-</script>
-
 <template>
-  <header>
-    <img alt="Vue logo" class="logo" src="@/assets/logo.svg" width="125" height="125" />
-
-    <div class="wrapper text-tth-500 kamran">
-      <HelloWorld msg="You did it!" />
-
-      <nav>
-        <RouterLink to="/">Home</RouterLink>
-        <RouterLink to="/about">About</RouterLink>
-      </nav>
-    </div>
-  </header>
-
-  <RouterView />
+   <router-view></router-view>
+  <!-- <HomeCustomer /> -->
+   <!-- <ForgotPassword />  -->
+  <!-- <Login /> -->
+  <!-- <CreateAccount /> -->
+  <!-- <ul class="post-grid">
+    <li v-for="(post, index) in filteredPosts" :key="index">
+      <h3>{{post.title}}</h3>
+      <template v-if="countWords(post.desc) > 21">
+        <p>
+          <span v-if="!post.readActivated">
+          <strong>{{post.desc.split(' ').slice(0,14).join(' ')}}...</strong>
+          <i>{{post.desc.split(' ').slice(14, post.desc.length).join('')}}</i>
+          </span>
+          <span v-else>{{post.desc}}</span>
+          <span class="read-more-label" v-if="countWords(post.desc) > 21" @click="post.readActivated = !post.readActivated">{{!post.readActivated ? 'Read More' : 'Read Less'}}</span>
+        </p>
+      </template>
+      <template v-else>
+        <p>{{post.desc}}</p>
+      </template>
+    </li>
+  </ul> -->
 </template>
 
-<style scoped>
-header {
-  line-height: 1.5;
-  max-height: 100vh;
+<script>
+// import HomeCustomer from './components/HomeCustomer.vue';
+// import CreateAccount from './components/CreateAccount.vue';
+// import Login from './components/Login.vue';
```

Comment:
Please remove this commented code,

Reply:
Commented code has been removed from all the pages